tests/unit/modules/dcnm/fixtures/dcnm_intf_vpc_configs.json

```diff
@@ -354,10 +351,8 @@
 				"peer2_pcid": 2
 			},
 			"switch": [
-				[
 					"10.122.84.109",
-			"10.122.84.108"
-				]
+			    "10.122.84.108"
```

Comment:
Here and elsewhere update to use a private ip address.

Reply:
Ok. Will modify the code to use private IP addresses